### PR TITLE
postgres user step added

### PR DESCRIPTION
**Pull request by juggernaut451**

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -30,15 +30,23 @@ Follow this guide to setup your development machine.
...
     ```
+
+6. Creating user
+
+    ```
+    sudo -u postgres createuser [user] -s --pwprompt
+
+    ```
     
 6. Change credential in setting/dev.py
```
Comment (vnbang2003) [inline docs/setup.md]: Can you fix the serial number ?

**Comment by juggernaut451** [comment]
@vnbang2003 done

File: docs/setup.md [hunk 2]
```diff
@@ -30,25 +30,33 @@ Follow this guide to setup your development machine.
     ```
     cp settings/dev.sample.py settings/dev.py
     ```
+
+6. Creating user
```
Comment (vnbang2003) [inline docs/setup.md]: It should be Creating postgres user ?
We need to put point why they need this step.

Comment (juggernaut451) [inline docs/setup.md]: They need this as its hard to setup project for new user. Our documentation should be so brief that no one should be stuck in between while setting up the project

**Pull request closed without merging**